sympa.spec

```diff
@@ -614,6 +614,7 @@ ln -s %{_datadir}/doc/%{name}/README \
 ln -s %{_datadir}/doc/%{name}/README \
     %{buildroot}/%{_datadir}/sympa/default/README
 %endif
+mv %{buildroot}%{_sysconfdir}/sympa/sympa.conf-dist __doc/
```

Comment:
I'm fine with this line going in the rpm specfile, but imho this is wrong that sympa installs this file in sysconfdir. This clearly is a doc file, both @racke and I agreed on that when the PR was discussed on the mailing list.